fabfile.py

```diff
@@ -18,12 +18,17 @@
 from fabric.api import run
 from fabric.contrib import files
 
+from secrets import SECRETS
+
+VOLONTULO_SENTRY_DSN = SECRETS.VOLONTULO_SENTRY_DSN
```

Comment:
what's the point of doint that?

Reply:
Importing from secrets - if you have better solution, feel free to add comments.